var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -76,6 +79,7 @@ def configure_args(self):
             args.append('--enable-netloc')
 
         args.extend(self.enable_or_disable('cairo'))
+        args.extend(self.enable_or_disable('nvml'))
```

Comment:
available with `hwloc` 1.7+ from what I found, so should be save to add like this :)